cl/citations/management/commands/cl_find_citations_recap.py

```diff
@@ -13,28 +13,41 @@
 from cl.lib.celery_utils import CeleryThrottle
 from cl.lib.command_utils import VerboseCommand
 from cl.lib.types import OptionsType
-from cl.search.models import Opinion
+from cl.search.models import Opinion, RECAPDocument
+
+
+def _parse_args_list(args: str) -> list:
+    return [s.strip() for s in args.split(",")]
```

Comment:
I think the way to normally do this is to do nargs="*", and then to run the script like:

```
--doc-pks 1 2 3
```

It looks like you're inventing a CSV parser, which is fun and all, but it's not normal, so I'd urge us to just do boring things.

Reply:
So that way works even if `type` is set to `int` for the arg? See, e.g., https://github.com/freelawproject/courtlistener/blob/main/cl/citations/management/commands/cl_find_citations.py#L25